storefront/src/app/core/config/config-flags.service.ts

```
@@ -19,6 +19,19 @@ export class ConfigFlagsService {
             return;
         }
         try {
+            if (subdomain === 'admin') {
+                const response = await firstValueFrom(
+                    this.#http.get<ApiResponse<Record<string, string>>>(
+                        `${environment.apiBaseUrl}/platform/cms/config`
+                    )
+                );
+                if (response.result === 'SUCCESS' && response.data) {
+                    this.#flags.clear();
+                    Object.entries(response.data).forEach(([k, v]) => this.#flags.set(k, v));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)